server/pulp/server/db/fields.py

```diff
@@ -48,3 +50,16 @@ def to_python(self, value):
         """
         ret = super(UTCDateTimeField, self).to_python(value)
         return dateutils.ensure_tz(ret)
+
+    def _validate(self, value, **kwargs):
```

Comment:
We should use `clean()` instead of `_validate()`. The docs say `clean()` is good for cleaning and pre-validation and it's part of the public API.

http://docs.mongoengine.org/en/latest/guide/document-instances.html#pre-save-data-validation-and-cleaning